github/repos.go

```diff
@@ -1015,6 +1017,8 @@ type ProtectionRequest struct {
 
 // RequiredStatusChecks represents the protection status of a individual branch.
 type RequiredStatusChecks struct {
+	URL         *string `json:"url,omitempty"`
```

Comment:
Now that I'm looking at this a little more closely, let's please move these two new fields to the end of the struct and make `URL` the last field.  Thanks.

Reply:
Done